index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header-section">
+    <div class="bg-image"></div>
+    <nav>
+        <img src="images/menu.svg" alt="">
+    </nav>
+  </header>
+  <main id="section-main">
+    <section>
+        <h1>AMBROSE <br> ISIGBA</h1>
+        <p>Hello! I am a passionate software developer! my goal
+         is to help you build a quality product(s), features 
+         and/or website with international standard. You are 
+         highly invited to take a tour of my works. If you fall in 
+          love with  my work gallery, and you are planning for one or 
+         you need a code mentor. AMBROSE's CODELAB get you covered</p>
+      </section>
+    </main>
+  <form>
+     <div id="second-main">
+      <button type="button">Start colaboration</button>
+      </div>
```

Comment:
- [ ] Please make sure your hover and focus effects follow the Figma design template. Right now your hover effect looks like this
> Your Design
![image](https://user-images.githubusercontent.com/90556221/170482503-e88795dd-c5bd-44bb-8e48-50717e4fd1c3.png)

But the Figma template has specified the following
> Figma Design
![image](https://user-images.githubusercontent.com/90556221/170482590-ef3a9c53-fe92-4ca0-8c67-29d1d703ad39.png)